package.json

```diff
@@ -1,25 +1,50 @@
 {
   "name": "surge-github-autorelease",
-  "version": "1.0.22",
-  "description": "",
-  "main": "sg-auto-release.js",
+  "version": "1.0.23",
+  "description": "Auto version and release for surge and github comments",
+  "license": "UNLICENSED",
+  "author": {
+    "name": "Ronen Stern",
+    "email": "ronens@wix.com"
+  },
+  "main": "dist/src/index.js",
+  "files": [
+    "dist"
+  ],
   "bin": {
-    "surge-github-autorelease": "main.js"
+    "surge-github-autorelease": "dist/src/cli.js"
+  },
+  "publishConfig": {
+    "registry": "http://npm.dev.wixpress.com/"
   },
   "scripts": {
-    "build": ":",
-    "test": ":",
-    "release": "wnpm-release",
-    "postpublish": "rm -f npm-shrinkwrap.json"
+    "start": "yoshi start",
+    "precommit": "yoshi lint",
+    "pretest": "yoshi build",
+    "test": "yoshi test",
+    "posttest": "yoshi lint",
+    "release": "yoshi release"
   },
   "devDependencies": {
-    "wnpm-ci": "latest"
```

Comment:
Was this removed by intention? How would it update the version then?

Reply:
Ummmm... 🤷‍♂️

Reply:
wnpm-ci is responsible to bump versions automatically so you don’t need to bump versions on your own.

Add it back to the devDependencies and don’t change the version manually

Reply:
Right, does it matter if it's before or after `yoshi release`? Or instead?

Reply:
It looks like yoshi is doing it for us, so actually we don’t need it.
https://github.com/wix/yoshi/blob/master/packages/yoshi/src/commands/release.js
Good to know